test/fixtures/hello-cloud/daemon_set.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: busybox
+spec:
+  template:
+    metadata:
+      labels:
+        app: busybox
+    spec:
+      containers:
+      - name: busybox
```

Comment:
nit: Doesn't make a difference since this won't come close to hitting 40s, but I'd use nginx here like we do to emulate web, since Daemonsets aren't expected to ever complete.